sources/editor/Stride.GameStudio/Helpers/AvalonDockHelper.cs

```diff
@@ -127,9 +126,7 @@ private static void ElementAdded(object sender, LayoutElementEventArgs e)
         /// </remarks>
         private static void AdjustAnchorableHideAndCloseCommands(LayoutAnchorable anchorable)
         {
-            var layoutItem = (LayoutAnchorableItem)anchorable.Root.Manager.GetLayoutItemFromModel(anchorable);
-            if (layoutItem == null)
-                throw new InvalidOperationException("The anchorable must be added to the docking manager before calling this method.");
+            var layoutItem = (LayoutAnchorableItem)anchorable.Root.Manager.GetLayoutItemFromModel(anchorable) ?? throw new InvalidOperationException("The anchorable must be added to the docking manager before calling this method.");
```

Comment:
Might wanna split it in two

Reply:
Right, definitely too long a line of code